react/components/SelectionListOrderBy.js

```diff
@@ -66,8 +66,8 @@ class SelectionListOrderBy extends Component {
             return (
               <div
                 key={opt.label}
-                className={classNames('vtex-orderby__item pointer pv3 ph7 bb b--light-gray', {
-                  'vtex-orderby__item--active dark-gray normal': active,
+                className={classNames('vtex-orderby__item pointer pv3 ph7 bb b--muted-4', {
+                  'vtex-orderby__item--active t-body c-on-base': active,
                   'fw3': !active,
```

Comment:
It is missing.

Reply:
I've replaced it with a c-disabled class, since we don't have classes for font-weight in design tokens.